Composite/Plugins/Functions/WidgetFunctionProviders/StandardWidgetFunctionProvider/DataReference/PageReferenceSelectorWidgetFunctionBase.cs

```diff
@@ -43,15 +52,22 @@ public override XElement GetWidgetMarkup(ParameterList parameters, string label,
         private XElement GetSearchToken(ParameterList parameters, XElement selector)
         {
             var parameter = GetParameterElement(parameters);
-            if (parameter == null)
+            if (parameter == null || !bool.TryParse(parameter.ToString(), out bool result) || !result)
                 return null;
 
             var f = Namespaces.BindingFormsStdFuncLib10;
+            var ft = Namespaces.Function10;
+
             var element = new XElement(selector.Name.Namespace + "DataReferenceTreeSelector.SearchToken",
                 new XElement(f + "StaticMethodCall",
                     new XAttribute("Type", TypeManager.SerializeType(typeof(PageReferenceSelectorWidgetFunctionBase))),
                     new XAttribute("Method", nameof(GetPageSearchToken)),
-                    new XElement(f + "StaticMethodCall.Parameters", parameter)));
+                    new XElement(f + "StaticMethodCall.Parameters",
+                        new XElement(ft + "function",
+                            new XAttribute("name", "Composite.Pages.GetPageId"),
+                            new XElement(ft + "param",
+                                new XAttribute("name", "SitemapScope"),
```

Comment:
nameof(SitemapScope)

Reply:
Thanks, updated, also updated in GetPageIdFunction class